### ISSUE TITLE
1.3.4 vocabulary emphasis

### ISSUE THREAD

**Issue by JonathanKatz91** [w3c/wcag21#320]
For 1.3.4 it also needs to be emphasized that vocabulary should be understood - this is for accessibility to people with cognitive disabilities.

**Comment by awkawk** [comment]
@JonathanKatz91 Can you clarify? I'm not sure if you are clarifying the existing text or suggesting an additional aspect to the SC.

**Comment by JonathanKatz91** [comment]
I am suggesting additional content for the SC – vocabulary should consist of words that are readily understood by a wide audience wherever possible.

**Comment by jake-abma** [comment]
@JonathanKatz91 as this is indeed one of the reasons for this SC would you be fine with it when we take this into account when writing the Understanding doc?

**Comment by JonathanKatz91** [comment]
Yep! As long as it is clear.

**Comment by jake-abma** [comment]
@JonathanKatz91  ok, thank you, we'll have the comment noted for the Understanding doc.

**Comment by awkawk** [comment]
Thank you for your comment. Adding additional requirements for text within content to use a constrained vocabulary was covered in a separate SC that was not ultimately able to be incorporated into the drafts at this stage. We have an issue that speaks to this that is deferred for future consideration.
